Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishHolidayParserConfiguration.cs

```diff
@@ -19,41 +19,110 @@ public EnglishHolidayParserConfiguration() : base()
+                {"girlsday", GirlsDay},
+                {"whiteloverday", WhiteLoverDay},
+                {"loverday", ValentinesDay},
+                {"christmas", ChristmasDay},
+                {"xmas", ChristmasDay},
+                {"newyear", NewYear},
+                {"newyearday", NewYear},
+                {"newyearsday", NewYear},
+                {"inaugurationday", InaugurationDay},
+                {"groundhougday", GroundhogDay},
+                {"valentinesday", ValentinesDay},
+                {"stpatrickday", StPatrickDay},
+                {"aprilfools", FoolDay},
+                {"stgeorgeday", StGeorgeDay},
+                {"mayday", Mayday},
+                {"cincodemayoday", CincoDeMayoday},
+                {"baptisteday", BaptisteDay},
+                {"usindependenceday", USAIndependenceDay},
+                {"independenceday", USAIndependenceDay},
+                {"bastilleday", BastilleDay},
+                {"halloweenday", HalloweenDay},
+                {"allhallowday", AllHallowDay},
+                {"allsoulsday", AllSoulsday},
+                {"guyfawkesday", GuyFawkesDay},
+                {"veteransday", Veteransday},
+                {"christmaseve", ChristmasEve},
+                {"newyeareve", NewYearEve},
             };
         }
         
-        private static DateObject Yuandan(int year) => new DateObject(year, 1, 1);
-        private static DateObject ChineseNationalDay(int year) => new DateObject(year, 10, 1);
-        private static DateObject WorkDay(int year) => new DateObject(year, 5, 1);
+        private static DateObject NewYear(int year) => new DateObject(year, 1, 1);
+        private static DateObject NewYearEve(int year) => new DateObject(year-1, 12, 31);
         private static DateObject ChristmasDay(int year) => new DateObject(year, 12, 25);
-        private static DateObject LoverDay(int year) => new DateObject(year, 2, 14);
+        private static DateObject ChristmasEve(int year) => new DateObject(year, 12, 24);
+        private static DateObject ValentinesDay(int year) => new DateObject(year, 2, 14);
         private static DateObject WhiteLoverDay(int year) => new DateObject(year, 3, 14);
-        private static DateObject ChineseMilbuildday(int year) => new DateObject(year, 8, 1);
         private static DateObject FoolDay(int year) => new DateObject(year, 4, 1);
         private static DateObject GirlsDay(int year) => new DateObject(year, 3, 7);
         private static DateObject TreePlantDay(int year) => new DateObject(year, 3, 12);
         private static DateObject FemaleDay(int year) => new DateObject(year, 3, 8);
         private static DateObject ChildrenDay(int year) => new DateObject(year, 6, 1);
         private static DateObject YouthDay(int year) => new DateObject(year, 5, 4);
         private static DateObject TeacherDay(int year) => new DateObject(year, 9, 10);
-        private static DateObject AllSaintsDay(int year) => new DateObject(year, 10, 31);
```

Comment:
It cannot support the resolution of Easter day and Chinese New year now. In Chinese holiday parser, we will return Lunar flag for Chinese new year. But for Easter day, we don't have a solution yet.